packages/nauth0/src/server/routes/callback.ts

```diff
@@ -28,9 +29,15 @@ export const callbackRoute: NAuth0ApiRoute = async (req, res, opts) => {
     path: '/',
   });
 
+  const redirect = decodeState(state)[RedirectKey] as string;
+
   res
     .writeHead(302, {
-      Location: '/', // TODO: Get the redirectUri
+      Location: redirect
+        ? redirect
+        : opts.postLoginRedirectUri
+        ? opts.postLoginRedirectUri
+        : '/',
```

Comment:
```suggestion
      Location: redirectUri ?? opts.postLoginRedirectUri ?? '/'
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Done.